indico/htdocs/sass/base/_pages.scss

```diff
@@ -215,24 +215,14 @@
 
     header h2 {
         color: $orange;
-        font-size: 2em;
         font-weight: normal;
```

Comment:
I think we might want to switch to bold now. Looks a bit too thin now that it's smaller. Removing this rule should be enough.